examples/notebook/tensorflow/alexnet_mnist/inc_quantize_model.py

```diff
@@ -48,22 +48,30 @@ def compare_ver(src, dst):
         return -1
     return 0
 
-def auto_tune(input_graph_path, config, batch_size, int8_pb_file):
+def auto_tune(input_graph_path, batch_size):
     dataset = Dataset()
-    dataloader = DataLoader(framework='tensorflow', dataset=dataset, batch_size = batch_size)
+    dataloader = DataLoader(framework='tensorflow', dataset=dataset, batch_size=batch_size)
+    tuning_criterion = TuningCriterion(max_trials=100)
+    config = PostTrainingQuantConfig(approach="static", tuning_criterion=tuning_criterion,
+                                     accuracy_criterion = AccuracyCriterion(
+                                         higher_is_better=True, 
+                                         criterion='relative',  
+                                         tolerable_loss=0.01  )
+                                    )
     q_model = fit(
         model=input_graph_path,
         conf=config,
         calib_dataloader=dataloader,
         eval_dataloader=dataloader)
+    
 
     return q_model
 
 
-config = PostTrainingQuantConfig()
+#yaml_file = "alexnet.yaml"
```

Comment:
rm this line, useless.

Reply:
Thank you for letting me know.. i will do the necessary changes.

Reply:
Done all the necessary changes.